### DISCUSSION TITLE
Yeet everything except cdbot

### DISCUSSION THREAD

**Discussion by ghost** [CyberDiscovery/cyberdisc-bot#599] [Ideas]
we don't need no corporate bots in our ends/

**Comment by thebeanogamer** [comment]
Duplicate of #294

**Reply by ghost** [reply]
might give another crack at auto mod actually, simple regex for words and invites, check if more than x mentions. And for anti spam, apparently you can apply cool downs to on_message.

**Reply by ghost** [reply]
last time well, I knew about as much python as the server has braincells
